nbs/07_clean.ipynb

```diff
@@ -168,6 +170,42 @@
     "               'source': 'awesome_code'})"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tst2 = {'cell_type': 'code',\n",
+    "       'execution_count': 26,\n",
+    "       'metadata': {'tags':[]},\n",
+    "       'outputs': [{'execution_count': 2, \n",
+    "                    'data': {\n",
+    "                        'application/vnd.google.colaboratory.intrinsic+json': {\n",
```

Comment:
There's still a lot of data here in the test. Can you minimize it to what's needed to test the logic?

Reply:
Let' me know if that's a better test